bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx

```diff
@@ -26,6 +26,12 @@ The VNC protocol is platform-independent, supporting Linux, Windows, and macOS,
 - A [Mac mini](/bare-metal/apple-silicon/how-to/create-mac-mini/)
 
 ## Accessing the remote desktop
+
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```

Comment:
I think something is missing after "not" in this sentence "and does not the default VNC port 5900"

Reply:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```